scripts/idl/README.md

```diff
@@ -97,6 +97,16 @@ server cluster AccessControl = 31 {
   // attributes may be read-only as well
   readonly attribute int16u clusterRevision = 65533;
 
+  // Code-gen specific information for storage can be added as a keyword list
+  // after the attribute number. Specifically these are supported
+  //   - `callback` is the equivalent of EXTERNAL in ember/zap, which means
```

Comment:
Should we indicate NVM vs not for the RAM case?

Reply:
Is nvm used currently? If yes I will need to come up with some keyword for it too. What is more common nvm or ram? I want the common one to be the default.

Reply:
Added a `persist` keyword.